File: Boeing/FizzBuzz/FizzBuzz.c
#include<stdio.h>

// Write a C program that prints the numbers 1 to 100. For multiples of three print the string “Fizz” instead of the number.
// For multiples of five print the string “Buzz” instead of the number.
// For multiples of both three and five, print the string FizzBuzz instead of the number.

void FizzBuzz(int n) {
    for (int i = 1; i <= n; i++) {
        if (i % 3 != 0 && i % 5 != 0) {
            printf("%d\n", i);
            continue;
        }
        if (i % 3 == 0) {
            printf("Fizz");
        }
        if (i % 5 == 0) {
            printf("Buzz");
        }
        printf("\n");
    }
}

int main() {
    FizzBuzz(100);
    return 0;
}

File: Boeing/FizzBuzz/FizzBuzz.py

def FizzBuzz(n: int) -> str:
    '''
    Write a C program that prints the numbers 1 to 100. For multiples of three print the string “Fizz” instead of the number.
    For multiples of five print the string “Buzz” instead of the number.
    For multiples of both three and five, print the string FizzBuzz instead of the number.
    '''
    for i in range(1, n + 1):
        s = ''
        if i % 3 == 0:
            s = s + 'fizz'
        if i % 5 == 0:
            s = s + 'buzz'
        if i % 3 != 0 and i % 5 != 0:
            s = str(i)
        yield s

if __name__ == '__main__':
    for k in FizzBuzz(100):
        print(k)
